tests/src/test/scala/com/nvidia/spark/rapids/BroadcastHashJoinSuite.scala

```diff
@@ -52,17 +55,21 @@ class BroadcastHashJoinSuite extends SparkQueryCompareTestSuite {
 
       for (name <- Seq("BROADCAST", "BROADCASTJOIN", "MAPJOIN")) {
         val plan1 = spark.sql(s"SELECT /*+ $name(t) */ * FROM t JOIN u ON t.longs = u.longs")
-          .queryExecution.executedPlan
         val plan2 = spark.sql(s"SELECT /*+ $name(u) */ * FROM t JOIN u ON t.longs = u.longs")
-          .queryExecution.executedPlan
 
-        val res1 = plan1.find(ShimLoader.getSparkShims.isGpuBroadcastHashJoin(_))
-        val res2 = plan2.find(ShimLoader.getSparkShims.isGpuBroadcastHashJoin(_))
+        val finalplan1 = plan1.queryExecution.executedPlan
+        plan1.collect()
+        val finalPlan1 = findOperator(finalplan1,
+          ShimLoader.getSparkShims.isGpuBroadcastHashJoin(_))
```

Comment:
nit: the `(_)` is redundant here and can be removed so that we just pass a method reference.

Reply:
for example: `val finalPlan1 = findOperator(finalplan1, ShimLoader.getSparkShims.isGpuBroadcastHashJoin)`

Reply:
done